docs/getting-started-guides/coreos/bare_metal_calico.md

```diff
@@ -112,10 +148,26 @@ Move the modified `node-config.yaml` to your Kubernetes node machine and install
 sudo coreos-install -d /dev/sda -C stable -c node-config.yaml
 ```
 
-Once complete, eject the bootable disc and restart the server.  When it comes back up, you should have SSH access as the `core` user using the public key provided in the `node-config.yaml` file.  It will take some time for the node to be fully configured.  Once fully configured, you can check that the node is running with the following command on the Kubernetes master.
+Once complete, eject the bootable disc and restart the server.  When it comes back up, you should have SSH access as the `core` user using the public key provided in the `node-config.yaml` file.  It will take some time for the node to be fully configured.
+
+## Connectivity Outside the Cluster
+To setup Internet Connectivity for your containers, you may need to program the NAT rules described in our [Ubuntu Guide](https://github.com/kubernetes/kubernetes/blob/master/docs/getting-started-guides/ubuntu-calico.md#nat-on-the-nodes).
+
+## Configure Your Workstation
```

Comment:
I'd change this title to something more descriptive.

"Configure Kubectl Access" or something.